core/src/main/scala/org/apache/spark/util/collection/BitSet.scala

```diff
@@ -58,6 +58,20 @@ class BitSet(numBits: Int) extends Serializable {
   }
 
   /**
+   * Clear all the bits up to a given index
+   */
+  def clearUntil(bitIndex: Int) {
```

Comment:
I'd declare this as `...): Unit = {` and fix `setUntil` above while here. It's technically a non-private class so want to be careful about the API. Hopefully MiMa doesn't tell us we've accidentally got a method here with non-void return type.